solutions/digital_root_solution.rb

```diff
@@ -1,3 +1,10 @@
+# Write a method, `digital_root(num)`. It should Sum the digits of a positive
+# integer. If it is greater than 10, sum the digits of the resulting number.
+# Keep repeating until there is only one digit in the result, called the
+# "digital root". **Do not use string conversion within your method.**
+#
```

Comment:
Replace the "Do not use string conversion" with the following note instead (make sure to update in both problem and solution file):

" **Do NOT use the built-in `Integer#to_s`, or `Integer#digits` methods in your implementation.**"